applications/FluidDynamicsApplication/custom_utilities/solver_settings.h

```diff
@@ -118,12 +119,18 @@ class SolverSettings
     virtual void SetStrategy(StrategyLabel const& rStrategyLabel,
                              typename TLinearSolver::Pointer pLinearSolver,
                              const double Tolerance,
-                             const unsigned int MaxIter) = 0;
+                             const unsigned int MaxIter) {}
```

Comment:
The function was abstract to make sure it is overriden in derived classes. What is the technical reason for changing this?

I am not opposed to the change, but I would recommend at least throwing a runtime error for the base class implementation.

Reply:
I realized that it can be done with the pure virtual and no override. Thank you for the suggestion. 
I changed it now.